assets/pages/subscriptions-landing/components/featuredProductAb.jsx

```diff
@@ -83,7 +82,7 @@ function FeaturedProductAb(props: PropTypes) {
       cta={getCta(product)}
       headingSize={headingSize}
       product={product.name}
-      hasTimer={flashSaleIsActive(product.name, countryGroupId)}
+      hasTimer={false}
```

Comment:
Here's a function I played around with that works but is a hack in case it's useful as somewhere to start later on

```
function showTimer(product: Product, countryGroupId: countryGroupId): boolean {
  if(flashSaleIsActive(product.name, countryGroupId)) {
    const now = new Date();
    const endTime = getEndTime(product.name, countryGroupId);
    const sevenDaysFromNow = now.setDate(now.getDate() + 7);

    return sevenDaysFromNow - endTime > 0;
  }
  return false;
}
```

Reply:
There is now a function to do this added in https://github.com/guardian/support-frontend/pull/1455